.gitignore

```diff
@@ -1 +1,4 @@
 node_modules/
+map.js
+style.css
+index.html
```

Comment:
You should remove these files from the gitignore, otherwise changes to them in the future may not be picked up by git. As soon as that's fixed I'll merge your changes :)